Source/OIDAuthorizationRequest.m

```diff
@@ -105,6 +105,20 @@ - (instancetype)init
                    additionalParameters:)
     )
 
+/*! @brief Check if the response type is one AppAuth supports
+    @remarks AppAuth only supports the `code` and `code id_token` response types.
+    @see https://github.com/openid/AppAuth-iOS/issues/98
+    @see https://github.com/openid/AppAuth-iOS/issues/292
+ */
++ (BOOL)isSupportedResponseType:(NSString *)responseType
+{
+  NSString *codeIdToken = [@[OIDResponseTypeCode, OIDResponseTypeIDToken]
+      componentsJoinedByString:@" "];
+
+  return [responseType isEqualToString:OIDResponseTypeCode] ||
```

Comment:
Put the `||` on the next line, aligned with the `[`

Reply:
Fixed.